actions/hosts.last_checkin.yaml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
final comment: add an entry to the CHANGELOG.md and increment the version in pack.yaml